### ISSUE TITLE
Clean up the fields in the Recording admin edit form

### ISSUE THREAD

**Issue by jgonggrijp** [UUDigitalHumanitieslab/microcontact#68]
The admin form now has nice fieldsets separating uploader information from recording/recordee information. However, at an earlier stage while working on #11 and #65, I also added verbose names to some of the fields in order to clarify their intent. As a consequence, this information is now double represented in the admin form for some fields but not for others. This is a bit ugly, but not urgent enough to fix it now.

Basically an outtake from #48.
